pyvlx/api/frames/frame_activate_scene.py

```diff
@@ -26,24 +27,26 @@ def __init__(
         self.priority = Priority.USER_LEVEL_2
         self.velocity = velocity
 
-    def get_payload(self):
+    def get_payload(self) -> bytes:
         """Return Payload."""
+        assert self.session_id is not None
+        assert self.scene_id is not None
```

Comment:
There are various places where there are assertions like this. Mostly they're there to reduce the possible types and I _think_ in at least most cases the code wouldn't have worked later in the function if the assertion failed anyways.